modules/richards/tests/gravity_head_2/tests

```diff
@@ -171,4 +171,43 @@
     rel_err = 1e-5
     abs_zero = 1e-4
   [../]
+
+  [./except01]
+    type = 'RunException'
+    input = 'gh01.i'
+    cli_args = 'Materials/rock/viscosity="1"'
+    expect_err = 'There are 2 Richards fluid variables, so you need to specify this number of viscosities, relperm_UO, seff_UO, sat_UO, density_UO, SUPG_UO'
+  [../]
+  [./except02]
+    type = 'RunException'
+    input = 'gh01.i'
```

Comment:
Ha, i just pushed a PR with some input files that didn't contain [Outputs].  Now i've decided go revert to basically-the-previous-version because: (1) i'm just adding more almost-useless files to the repo; (2) i now can't inspect the outputs if i do need to in the future.

I'm about to PR this "basically-the-previous-version" ( each RunException now writes to its own unique file).